src/components/UniversalForm/types.tsx

```diff
@@ -11,6 +11,7 @@ export type FormProps = {
    action: string;
    method: METHOD;
    inputs: InputElementProps[];
+   submitNavigation?: string;
```

Comment:
please explain why do you need this and why the name is so weird

Reply:
that row is 'artificial'. Deleted